bin/vint

```diff
@@ -17,9 +17,9 @@ from pathlib import Path
 #   $ python -m unittest discover test
 
 if __name__ == '__main__':
-    vint_dir = Path(__file__).resolve().parent.parent
+    vint_root = Path(__file__).resolve().parent.parent
+    sys.path.append(str(vint_root))
 
-    sys.path.append(str(vint_dir))
     import vint
 
-    vint.main()
+    sys.exit(vint.main())
```

Comment:
What purpose of this change?

And I think `sys.exit` should take an integer as the exit code.
But `vint.main` does not return the integer on the current implementation.